content/scout/integrations/environment/sysdig.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Integrate with Docker Scout using Sysdig
+description: scout, sysdig, integration, image analysis, environments, supply chain
+keywords: Integrate your runtime environments with Docker Scout using Sysdig
+---
+
+{{< include "scout-early-access.md" >}}
+
+The Sysdig integration enables Docker Scout to automatically detect the images
+you're using for your running workloads. Activating this integration gives you
+real-time insights about your security posture, and lets you compare your
+builds with what's running in production.
+
+## How it works
+
+The Sysdig Agent captures which images are being used to run your container
+workloads. Docker Scout integrates with the Sysdig API to discover the images
+in your cluster. Each Sysdig integration maps to an environment. When an image
+is detected in the cluster, that image gets assigned to the associated
+environment.
+
+The prerequisites for integrating Docker Scout with Sysdig are:
+
+- You must install the Sysdig Agent in the cluster that you want to integrate.
+  For more information, see [Sysdig docs](https://docs.sysdig.com/en/docs/installation/).
+- You must enable the runtime insights feature in Sysdig.
+
+Additionally, the images that you deploy must be analyzed by Docker Scout
+before they can be assigned to an environment. The Sysdig integration doesn't
+trigger an image analysis. To analyze images automatically, you can integrate
```

Comment:
**suggestion:** could we perhaps make it a bit more clearer here that runtime environment is not the same thing as registry integrations?

As in:
```
Additionally, the images that you deploy must already be known by Docker Scout through the registry integrations found in *link*. Sysdig only assigns environments to images. It doesn't analyse and push that data into Scout.
```

Something better worded, but that conveys more of this?